src/ctia/http/routes/observable.clj

```diff
@@ -51,16 +51,25 @@
     :query [params IndicatorsByObservableQueryParams]
     :path-params [observable_type :- ObservableType
```

Comment:
I think the previous paging params become meaningless, perhaps you should change them.